index.js

```diff
@@ -226,7 +226,7 @@ metalsmith
   );
 
 if (options.algolia.privateKey) {
-  log('Algolia indexing enabled');
+  log('Algolian indexing enabled');
```

Comment:
I... think that you should check your regexes before applying them. :expressionless: